core/src/avm2/globals.rs

```diff
@@ -478,23 +478,6 @@ pub fn load_player_globals<'gc>(
         script
     );
 
-    // package `flash.display`
-    avm2_system_class!(
-        graphics,
-        activation,
-        flash::display::graphics::create_class(activation),
-        script
-    );
-
-    // package `flash.geom`
-
-    // package `flash.text`
```

Comment:
This comment still applies to `flash::text::font` (the last remaining Rust `flash.text` class), so I don't think it should be removed.